ui/app/src/components/CreateSiteDialog.tsx

```diff
@@ -520,12 +520,7 @@ function CreateSiteDialog() {
     };
     if (site.sandboxBranch) params.sandboxBranch = site.sandboxBranch;
     if (site.blueprintFields) params.siteParams = site.blueprintFields;
-    //TODO# remove this when change to Api2
```

Comment:
What about API1 call which is still done for built in blueprints?

Reply:
Built in blueprints are still using api1 create site

Reply:
built in blueprint uses fn "createParams" and marketplace uses fn "createMarketplaceParams"